scripts/UpdateMetaModule.ps1

```diff
@@ -0,0 +1,82 @@
+[CmdletBinding()]
+param
+(
+    [Parameter(Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [string] $Path
+)
+
+Write-Output "Running $($MyInvocation.MyCommand.Name)"
+
+$nuspecPath = (Get-Childitem �Path $Path -Include *.nuspec -Recurse).FullName
```

Comment:
We should support multiple nuspecs and then iterate over them

Reply:
Change Include to Filter

Reply:
Done